upgrade_testing/data/upgrade

```diff
@@ -90,10 +90,22 @@ function main() {
         post_tests
         STATUS=$?
     fi
+    check_no_apt_errors
```

Comment:
I think we want to call this after every `do-release-upgrade`, in case we have more than one. So let's add it to `do_upgrade_and_maybe_reboot()`, after `do_normal_upgrade()`.